MinPlatformPkg/Include/Library/PlatformConfigCheckLib.h

```diff
@@ -0,0 +1,26 @@
+/** @file
+
+  Platform configuration check library for TestPointCheckLib
+
+  Copyright (c) Microsoft Corporation. All rights reserved
+
+**/
+
+#ifndef _PLATFORM_CONFIG_CHECK_LIB_H_
+#define _PLATFORM_CONFIG_CHECK_LIB_H_
+
+#include <Uefi.h>
+
+/**
+  This function peforms platform level checks at Exit Boot Services.
+
+  @retval     EFI_SUCCESS  Function has completed successfully.
+              Other        Function error indicates failure.
+**/
+EFI_STATUS
```

Comment:
Is this function meant to actually check that platform configuration meets expectations? Or is it for just dumping information?

Reply:
I designed this function as a single entry point for any platform specific Exit Boot Services code. It would be up to the platform implementation to check any necessary configurations.

Now for my planned platform implementation this function would only be dumping information. I'll rename the function to better reflect this functionality.

Reply:
Okay, that's what I was thinking. Maybe start with something like a "dump" function now and add a "check" function in the future.